### PR TITLE
Adding check for enable file log

### PR DESCRIPTION
**Pull request by hrajput89**
Added passing of file logging status variable to test through conftest

#### Fixes #.

- [x] Ran `flake8`
- [x] Ran `run_repo_tests.sh`

### CODE REVIEW

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py [hunk 1]
```diff
@@ -41,11 +41,15 @@ def test_mask_password_in_logs(params_from_base_test_setup, password):
...
     testserver = params_from_base_test_setup["testserver"]
     device_enabled = params_from_base_test_setup["device_enabled"]
+    enable_file_logging = params_from_base_test_setup["enable_file_logging"]
 
     num_cbl_docs = 500
     if sync_gateway_version < "2.0.0" and log_file is not None:
         pytest.skip('This test cannot run with sg version below 2.0 or File logging is not enabled.')
 
+    if not enable_file_logging:
+        pytest.skip("Cannot run test without file logs")
```
Comment (sridevi-15) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py]: The above if condition is checking to have file logging enabled but with with different parameter. Isn't same check?

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py [hunk 2]
```diff
@@ -102,11 +106,15 @@ def test_verify_invalid_mask_password_in_logs(params_from_base_test_setup, inval
...
     testserver = params_from_base_test_setup["testserver"]
     device_enabled = params_from_base_test_setup["device_enabled"]
+    enable_file_logging = params_from_base_test_setup["enable_file_logging"]
 
     num_cbl_docs = 50
     if sync_gateway_version < "2.0.0" and log_file is not None:
         pytest.skip('This test cannot run with sg version below 2.0 or File logging is not enabled.')
 
+    if not enable_file_logging:
+        pytest.skip("Cannot run test without file logs")
```
Comment (sridevi-15) [inline testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py]: same as above comment

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py (hunk 1 above)
Comment (hrajput89) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py]: actually in a way yes, they are 2 different parameters. Removed the second condition and merged it in 1st condition as it more cleaner.

File: testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py (hunk 2 above)
Comment (hrajput89) [reply testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py]: did same changes

**Pull request merged**